src/background/adblocker.js

```diff
@@ -230,6 +230,18 @@ export const setup = asyncSetup('adblocker', [
   ),
 ]);
 
+function resolveInjectionTarget(details) {
+  const target = { tabId: details.tabId };
+
+  if (details.documentId) {
+    target.documentIds = [details.documentId];
+  } else {
+    target.frameIds = [details.frameId];
```

Comment:
The old code had in injectStyles a fallback for details.frameId being undefined:

```
   if (frameId !== undefined) {
    target.frameIds = [frameId];
  } else {
    target.allFrames = true;
  }
```

Do we also have to keep this special case here?